app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div>
+  <div class="bg-primary nav">
+    <div class="d-flex justify-content-center align-items-center gap-5">
+      <div class="ms-4">
+        <%= link_to "Back", authenticated_root_path, class:'text-white' %>
+      </div>
+
+      <div class="ms-3">
+        <h3 class="text-white">Transactions</h3>
+      </div>
+
+      <div class="ms-2">
+        <% if user_signed_in? %>
+            <%= button_to 'Sign Out', destroy_user_session_path, method: :delete, class: "btn btn-primary" %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <div>
+
+    <div class="ms-3 mt-3 mb-3">
+      Icon: <img src="<%= @group.icon %>", alt="group-image", style="width:60px; height:60px;">
+    </div>
+    <div class="ms-3 mb-3">
+      <h6>Total Amount: <%= @group.expenses.sum(:amount) %> $</h6>
+    </div>
+
+    <div class="d-flex flex-column gap-2 mb-3 ms-3 me-3">
+      <% @expenses.order(created_at: :desc).each do |expense| %>
+        <div class="d-flex flex-column border border-primary p-1">
+          <div>Transaction Name: <%= expense.name %></div>
+          <div>Transaction Amount: <%= expense.amount %>$</div>
```

Comment:
- Hey, also, from the design, you don't need to specify the `Transaction Name:` and `Transaction Amount`
- Let's follow the design layout as much as we can

*your design*
![image](https://user-images.githubusercontent.com/59397159/234869596-c3710145-9258-4d50-9bc9-0d98ea75b52a.png)


*design template*
![image](https://user-images.githubusercontent.com/59397159/234868511-660288a6-1339-4fa0-9711-9129c7a28c4e.png)